### DISCUSSION TITLE
[잡담] 죄송합니다 ㅠㅠ

### DISCUSSION THREAD

**Discussion by ghost** [4Moyede/HexatonClass-01#110] [잡담]
제가 계절학기 시험 치느라 지금까지 과제를 하나도 못 했는데요...

일단 내일 13시에 시험 제출 끝나서 그때부터 과제를 시작해볼 생각입니다...

시간이 충분할지는 모르겠지만 내일 안에 마칠 수 있도록 노력해보겠습니다

**Comment by yoomambo** [comment]
@JaebeomHan 매번 늦으시면 살짝 곤란해요~ 저번에도 늦었는데....
뭐 계절 시험이라 이해는 합니다만, 본인의 선택에 의해 계절도 들은거니 앞으로는 책임감 있는 모습 부탁해요.
다른 친구들도 계절 들으면서 여러가지로 바쁜건 다들 똑같다고 생각합니다.

그리고 제목 쓸 때는 , [잡담] ~~~ 이렇게 [] 로 어떤 부분인지 알려주세요
